### PR TITLE
Add support for adding arbitrary labels to RStudio Workbench pods

### LINKED ISSUE #101: rstudio-workbench pod metadata labels
Is there a way to add a metadata label to the pod template using the rstudio-workbench helm chart values.yaml?
This is what i need the workbench pod to look like:
```
apiVersion: v1
kind: Pod
metadata:
  name: demo
  labels:
    aadpodidbinding: <aad_pod_identity_name>
```

### PR DESCRIPTION
**Pull request by atheriel**
This should enable users that have additional integrations (like the Azure AD feature discussed in #101) that need to be injected via pod labels.

Closes #101.

### CODE REVIEW

File: charts/rstudio-workbench/values.yaml [hunk 1]
```diff
@@ -234,6 +234,8 @@ pod:
   volumeMounts: []
   # -- podAnnotations is a map of keys / values that will be added as annotations to the rstudio-pm pods
```
Comment (colearendt) [inline charts/rstudio-workbench/values.yaml:235]:
```suggestion
  # -- pod.annotations is a map of keys / values that will be added as annotations to the rstudio-pm pods
```
holdover from a prior age 😅

**Review by colearendt — APPROVED** [review]
Looks awesome! 1 nit - maybe add an example to the `ci/complex-values.yaml` or somewhere, so that we are exercising this in CI / ensure it templates properly and satisfies linting?

**Comment by colearendt** [comment]
Oh! And probably worth adding a `NEWS.md` entry as well!

**Comment by atheriel** [comment]
@colearendt Should be all good now, provided the install test passes.

**Pull request merged**